hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/raft/RaftBackendStoreProvider.java

```diff
@@ -219,8 +260,16 @@ public void createSnapshot() {
         StoreCommand command = new StoreCommand(StoreType.GRAPH,
                                                 StoreAction.SNAPSHOT, null);
         RaftStoreClosure closure = new RaftStoreClosure(command);
-        this.context.node().submitAndWait(command, closure);
-        LOG.debug("Graph '{}' has writed snapshot", this.graph());
+        RaftClosure<?> future = this.context.node().submitAndWait(command,
+                                                                  closure);
+        if (future != null) {
```

Comment:
when the future may be null?

Reply:
Maybe not. Maybe it's defensive programming